relay-spans/Cargo.toml

```diff
@@ -13,8 +13,10 @@ publish = false
 chrono.workspace = true
 enumset = "1.0.4"
 once_cell.workspace = true
+opentelemetry-proto = { git = "https://github.com/open-telemetry/opentelemetry-rust", features = ["gen-tonic", "with-serde", "trace"] }
```

Comment:
The crate is available in [crates.io](https://crates.io/crates/opentelemetry), any reason to install it from git?

Reply:
need a new PR that's not released yet, asked here
https://github.com/open-telemetry/opentelemetry-rust/pull/1462#issuecomment-1924135271